aws/resource_aws_cognito_user_group_test.go

```diff
@@ -0,0 +1,220 @@
+package aws
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSCognitoUserGroup_basic(t *testing.T) {
+	poolName := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	groupName := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	updatedGroupName := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
```

Comment:
Do you mind prepending a unique fixed prefix to each name? e.g. `tf_acc_`? This is to make it easy/possible to cleanup any potentially leaked resources from the test and make it easier to identify those leaks in general.